docs/migration-guide/to-15.md

```diff
@@ -1,17 +1,24 @@
 # Migrating to 15.0.0
 
-This release contains significant and breaking changes that will help us modernize our code so that Stylelint remains free of security issues.
+This release contains significant or breaking changes.
 
 ## Significant change
 
-We deprecated [most of the rules that enforce stylistic conventions](../user-guide/rules.md#deprecated). You should remove these rules from your configuration object.
+We've deprecated [76 of the rules that enforce stylistic conventions](../user-guide/rules.md#deprecated), e.g. [`indentation`](../../lib/rules/indentation/README.md).
 
-We recommend:
+When we created these rules, pretty printers (like [Prettier](https://prettier.io/)) didn't exist. They now offer a better way to consistently format code, especially whitespace. Linters and pretty printers are complementary tools that work together to help you write consistent and error-free code.
 
-- using a pretty printer, like [Prettier](https://prettier.io/), alongside Stylelint
-- extending [our standard config](https://www.npmjs.com/package/stylelint-config-standard) in your configuration object
+By deprecating these rules, we can:
 
-Stylelint and pretty printers are complementary tools that work together to help you write consistent and error-free code. To help facilitate this, we've removed the [deprecated rules](../user-guide/rules.md#deprecated) from the latest version of [our standard config](https://www.npmjs.com/package/stylelint-config-standard). You can [extend](../user-guide/configure.md#extends) the config using:
+- focus on writing and maintaining rules that help you [avoid errors](../user-guide/rules.md#avoid-errors) and [enforce (non-stylistic) conventions](../user-guide/rules.md#enforce-conventions), both of which are unique to Stylelint
+- modernize our codebase, e.g. move to ESM so that we can update our dependencies and keep Stylelint secure for you
```

Comment:
[note] This is a significant change, and I suspect our users will be looking for some explanation why we made it. I've kept it short and not gone into technical issues, e.g. that standalone lint rules are problematic for whitespace as they tend to introduce issues into other rules. I've emphasised the benefits it'll bring in the long run.